### ISSUE TITLE
Whether to support non-image environment？

### ISSUE THREAD

**Issue by qimingRen** [PKU-Alignment/SafeDreamer#3]
Is it valid for state space environments without image representation?

**Comment by hdadong** [comment]
Yeah, you can the code of Online Safety-Reward Planning (OSRP) for low-dimensional state input:
`
python  SafeDreamer/train.py --configs osrp_vector --method osrp --run.script eval_only --run.from_checkpoint /xxx/checkpoint.ckpt --task safetygymcoor_SafetyPointGoal1-v0 --jax.logical_gpus 0 --run.steps 10000
`
